src/components/FlawAffects/useEditableAffects.ts

```diff
@@ -0,0 +1,96 @@
+import { ref, onUnmounted } from 'vue';
+
+import { clone } from 'ramda';
+
+import { flaw } from '@/composables/useFlawModel';
+import { useFlawAffectsModel } from '@/composables/useFlawAffectsModel';
+
+import type { ZodAffectType } from '@/types/zodAffect';
+import { matcherForAffect } from '@/utils/helpers';
+
+import { useAffectSelections } from './useAffectSelections';
+
+const { initialAffects } = useFlawAffectsModel();
+
+const { isAffectSelected, toggleAffectSelection } = useAffectSelections();
+
+export const affectsBeingEdited = ref<ZodAffectType[]>([]);
+export const affectValuesPriorEdit = ref<ZodAffectType[]>([]);
```

Comment:
Since this refactor introduces global state, maybe it makes sense to move all this logic to a proper `pinia` store, so the components don't directly mutate this refs, it should also help debugging with the VueDevTools.